data/remnant/remnant missions.txt

```diff
@@ -4634,12 +4628,412 @@ mission "Remnant: Expanded Horizons Storms 1"
+	"hull repair rate" 10000
+	"fuel consumption" 5
+	"fuel energy" 1
+	"hull energy" 5
+
+ship "_CME Timer Ship"
+	# The Timer Weapon causes this ship to continually lose hull points
+	# The _CME Timer: Generator outfit counteracts this until there is a coronal mass ejection
+	# sprite "ship/shuttle" # uncomment this for debugging
+	attributes
+		"heat dissipation" 0.01
+		"hull" 0.5
+		"mass" 10
+		"drag" 100
+		"automaton" 1
+		"fuel capacity" 10
+		"ion resistance" 10000000
+		"ramscoop" 6
+		"thrust" 0
+		"turn" 600
+	outfits
+		"Timer Weapon"
+		"_CME Timer: Generator"
+	gun 0 0
+
+
+
+mission "Remnant: Expanded Horizons Novas 1"
+	name "Wolf-Rayet Star Comparison"
+	description "Observe coronal mass ejection events at both Caeculus and Ildaria and return with the data to <destination>."
+	passengers 1
+	source
+		government "Remnant"
+	waypoint
+		system "Caeculus"
+	waypoint
+		system "Ildaria"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Expanded Horizons Storms 1: done"
+		random < 30
+	on offer
+		require "Research Laboratory"
+		conversation
+			`	You have barely started your meal at a cafe table when you notice Dawn waving at you from another table. You walk over and sit down to see what she needs. "Hello, Captain <last>! Would you be interested in going on another research expedition?"`
+			choice
+				`	"Of course! Where do you need to go?"`
+				`	"Sorry, I'm busy right now."`
+					defer
+
+			`	"This study will actually require an expedition to two different stars, but the first is quite close by: Caeculus. We may have to remain there for some time, but what we observe should be quite exciting.`
+			choice
+				`	"What exactly will we be observing?"`
+					goto massejection
+				`	"Exciting? I'm not sure I like the sound of that."`
+
+			`	"Oh, it will not be dangerous if we keep our distance. This should be far safer than the expedition to Sagittarius A*, and for this first star we'll be right next door to the capital in case of any emergencies. Afterward, we will need to journey to Ildaria, in Ancestral space, so help should likewise be available."`
+			choice
+				`	"Okay, you've convinced me. What will we be watching, anyway?"`
+					goto massejection
+
+			label massejection
+			`	Her eyes light up as she animately begins explaining "As you've probably noticed by now, the Ember Waste is filled with dead and dying stars, relics of an age long past. However, Remnant scientists have been puzzled for centuries by the fact that the age of these stars does not match the astrophysical models we brought over from Ancestral space. Based on those models, these stars are all far too young to produce novae, or too small to go supernova. So, we want to compare two stars at similar states in their stellar evolution: one inside the Ember Waste, and one in the wider galaxy. Because they usually only live for a few million years, and are physically distinct, we have chosen two Wolf-Rayet stars to serve as our baseline."`
```

Comment:
```suggestion
			`	Her eyes light up as she animately begins explaining "As you've probably noticed by now, the Ember Waste is filled with dead and dying stars, relics of an age long past. However, Remnant scientists have been puzzled for centuries by the fact that the age of these stars does not match the astrophysical models we brought over from Ancestral space. Based on those models, these stars are all far too young to produce novae, or too small to go supernova. So, we want to compare two stars at similar states in their stellar evolution: one inside the Ember Waste, and one in the wider galaxy. Because they usually only live for a few million years, and are physically distinct, we have chosen two Wolf-Rayet stars to serve as our baseline.`
```

Reply:
I'm not sure what's up with the Cinis Dust Lab mission, as I tested it multiple times and it appears to be working for me.

As for the waypoints, I've somewhat assumed the player will listen to Dawn's instructions as to which stars to visit first, which are placed in order of distance anyway. I don't think it's really a major issue, but if people really want those lines to be removed it would be an easy fix.